types/index.d.ts

```diff
@@ -306,6 +306,34 @@ export namespace invoices {
   function list(): Promise<ResultsResponse<Invoice>>;
 }
 
+export namespace session {
+  function get(): Promise<{ [key: string]: any }>;
+  function getCookie(): string;
+  function setCookie(): string;
+}
+
+export namespace functions {
+  function request(method: string, appId: string, functionName: string, data?: any, options?: any): Promise<any>;
+  function get(
+    appId: string,
+    functionName: string,
+    data?: any,
+    options?: any,
+  ): Promise<any>;
+  function put(
+    appId: string,
+    functionName: string,
+    data?: any,
+    options?: any,
+  ): Promise<any>;
+  function post(
+    appId: string,
+    functionName: string,
+    data?: any,
+    options?: any,
+  ): Promise<any>;
```

Comment:
delete?

Reply:
Can't seem to use the word "delete" in a namespace like this. I tried a couple other approaches with no luck, then realized we also left `delete` out of the types for other namespaces